modules/cstrike/cstrike/CstrikeNatives.cpp

```diff
@@ -1809,6 +1809,40 @@ static cell AMX_NATIVE_CALL cs_get_item_id(AMX* amx, cell* params)
 	return CSI_NONE;
 }
 
+// native bool:cs_get_item_alias(itemid, name[], name_maxlen, altname[] = "", altname_maxlen = 0); 
+static cell AMX_NATIVE_CALL cs_get_item_alias(AMX* amx, cell* params)
+{
+	if (ItemsManager.HasConfigError())
+	{
+		MF_LogError(amx, AMX_ERR_NATIVE, "Native cs_get_item_alias() is disabled because of corrupted or missing gamedata");
+		return 0;
+	}
+
+	auto itemid = params[1];
+
+	if (itemid == CSI_SHIELDGUN)
+	{
+		itemid = CSI_SHIELD;
+	}
+	else if (itemid == CSI_GLOCK)
+	{
+		itemid = CSI_GLOCK18;
```

Comment:
_GLOCK has index of 2, which is not used by game for some reason (likely the original HL glock) but you have some internal arrays used for alias which redirects _GLOCK data  to _GLOCK18 and "???" alias is associated to _GLOCK. I guess it doesn't hurt to add some fallback. I should probably add the constant with some comment.